roles/binary_bundler/tasks/deb_bundle.yml

```diff
@@ -1,13 +1,17 @@
 ---
-- name: Set Redpanda RPM and GPG Key details
+- name: Set Redpanda RPM
   set_fact:
     deb_link_actual: "{{ is_using_unstable | bool | ternary(deb_unstable_download_url, deb_download_url) }}"
+  tags:
+    - airgap-tarball-install
```

Comment:
airgap-tarball-install is an opt-in behavior? We need some readme updates on how/when this will be used. Generally I'm against setting tags because it doesn't make for clean, repeatable automation runs (e.g., if someone forgets to set it ... or accidentally sets it on one run).

Reply:
My assumption is that users probably don't want to reinstall redpanda every time they run the ansible playbooks and I don't have a good way to handle that behavior without also making upgrading challenging. Hence skip tags. If you're running ansible through CI it should be fairly straightforward to just add them as part of the invocation. 

Skip tags are also just generally useful for experimenting with the playbooks as we saw with PNC.

Reply:
Not sure I understand this comment. Ansible shouldn't be reinstalling redpanda on each run, if it is, that's broken behavior.

Reply:
So for the standard install it does indeed skip install if it's same version. For airgap via binary install I'd have to check redpanda binary exists, then with --version, then check whether our install version is higher. I'm planning to handle that in a patch pretty soon, but for now I've got the skip tags here.

Reply:
I've updated with version checking for the deb/rpm bundles. Latest is assumed to mean always install, otherwise we check and update if the version is lower than expected.